README.md

```diff
@@ -186,6 +186,10 @@ Although [kompose](https://github.com/kubernetes/kompose) can already convert do
 
 ## Advanced Usage
 
+### Dependency healthchecks
 If you require that an application is not started until one of its dependencies is healthy, you can add `condition: service_healthy` to the `depends_on`, and give the dependency a [Docker healthcheck](https://docs.docker.com/engine/reference/builder#healthcheck).
 
-Docker healthchecks are converted into [Readiness Probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/).
\ No newline at end of file
+Docker healthchecks are converted into [Readiness Probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/).
+
+### Running as specific users
+If any of the images you are running with kube-compose need to be run as a specific user, you can set the `--run-as-user` flag for `kube-compose up`. This will set each pod's `runAsUser`/`runAsGroup` based on either the [`user` property](https://docs.docker.com/compose/compose-file/#domainname-hostname-ipc-mac_address-privileged-read_only-shm_size-stdin_open-tty-user-working_dir) of its docker-compose service or the [`USER` configuration](https://docs.docker.com/engine/reference/builder/#user) of its Docker image (prioritising the former if both are present).
```

Comment:
Would also potentially mention how by default if an image is not specified with a `USER` it will run as root.